swagger/swagger.json

```diff
@@ -1926,11 +1930,13 @@
     },
     "Execution": {
       "required": [
+        "name",
```

Comment:
@ruescar  You are right about builds needing name and component (nice catch), however looking at the model, I don't executions need a name. This is what the title is for.

Reply:
True. Updated.